src/pages/Profile.jsx

```diff
@@ -84,6 +84,7 @@ const Profile = ({ members, overview, daoTokens, activities }) => {
             ens={profile?.ens}
             profile={profile}
             memberEntity={memberEntity}
+            refreshProfile={setProfile}
```

Comment:
This will not refresh the avatar address info unless the user goes to a new page